Cargo.toml

```diff
@@ -18,17 +18,21 @@ exclude = [".github/workflows/ci.yml",
 byteorder = {version = "1.3.4", default-features = false}
 rayon = "1.3.0"
 serde = {version = "1.0.106", optional = true}
-rand_core = "0.6"
+rand_core = "0.5"
 canonical = {version = "0.4", optional = true}
 canonical_derive = {version = "0.4", optional = true}
+dusk-bytes = {git="https://github.com/dusk-network/dusk-bytes", tag="v0.1.0"}
+
+# Required to fix the different version canonical_derive and dusk-bytes use for syn
+syn = "1.0.58"
```

Comment:
We should solve this. In dusk-bytes if possible.

Reply:
Fixed